datasets/ONS-UK-trade-in-services/main.py

```diff
@@ -1,58 +1,196 @@
 #!/usr/bin/env python
 # coding: utf-8
 # %%
+from gssutils import *
+import json
+
+cubes = Cubes("info.json")
+scraper_1 = Scraper('https://www.ons.gov.uk/businessindustryandtrade/' + \
+                  'internationaltrade/datasets/uktradeinservicesallcountriesnonseasonallyadjusted')
+scraper_1
+trace = TransformTrace()
+
+# %%
+info = json.load(open("info.json"))
+scraper_1.dataset.family = info["families"]
+
+# %%
+dist_1 = scraper_1.distributions[0]
+dist_1
+
+# %%
+first_url = dist_1.downloadURL
+first_url
+
+# %%
+tabs = {tab.name: tab for tab in scraper_1.distribution(latest=True, mediaType=Excel).as_databaker()}
+
+
+# %%
+tab = tabs["TiS by country"]
 
+# %%
+tab = tabs['TiS by country']
+datasetTitle = "ONS-UK-trade-in-services"
+columns = ["Period", "ONS Partner Geography", "Flow", "Trade Services", "Measure Type", "Unit", "Marker"]
+
+# %%
+trace.start(datasetTitle, tab, columns, first_url)
```

Comment:
replace line 38 with : 

`trace.start(datasetTitle, tab, columns, distribution.downloadURL)`

as the code defined above removes the defined variable holding the distribution download.

Reply:
Variable holding distribution download is removed and changes made to trace.start()